solid_utils/sampling.py

```diff
@@ -0,0 +1,133 @@
+import numpy as np
+import math
+import warnings
+from .variogram import remove_trend as rt
```

Comment:
Please use the absolute module import as `from solid_utils import variogram` or `from solid_utils.variogram import remove_trend`. I would recommend using `remove_trend` without abbreviating it as `rt`, as it's more readable.

Reply:
I use relative import just in case the ATBD directory is not added to the `PYTHONPATH`. @EJFielding mentioned it has been added so I will change it to absolute import.
I abbreviate `remove_trend` to `rt` since it conflicts with one function of `samp_pair`.
```
def samp_pair(x,y,data,num_samples=1000000,remove_trend=False)
...
```
If you have a better way to solve this conflict, please tell me.

Reply:
We could use `def samp_pair(deramp=False)`.

Reply:
Great idea. I will change it.